docs/Analyzer Configuration.md

```diff
@@ -108,6 +108,29 @@ Examples:
 |`dotnet_code_quality.null_check_validation_methods = NS.MyType.Validate(ParamType)` | Matches specific method 'Validate' with given fully qualified signature
 |`dotnet_code_quality.null_check_validation_methods = NS1.MyType1.Validate1(ParamType)~NS2.MyType2.Validate2(ParamType)` | Matches specific methods 'Validate1' and 'Validate2' with respective fully qualified signature
  
+### Excluded symbol names
+Option Name: `excluded_symbol_names`
+
+Configurable Rules: [CA1303](https://docs.microsoft.com/visualstudio/code-quality/ca1303-do-not-pass-literals-as-localized-parameters)
+
+Option Values: Names of symbols (separated by '~') that are excluded for analysis.
```

Comment:
Note that we cannot use the following as separators:
1. `;` or `#` token as these represent comment start tokens for .editorconfig file
2. `,` token as the documentation comment ID for method names must include comma separated parameter types

Reply:
Did you consider using bar | as a separator? (Similar to regular expressions separator.)

Reply:
Good suggestion, thanks I have addressed it.